tests/environment.py

```diff
@@ -492,6 +492,27 @@ class VirtualAPHostapd(VirtualAP):
     def __init__(self, radio: RadioHostapd, bssid: str):
         super().__init__(radio, bssid)
 
+    def get_ssid(self) -> str:
+        """Get current SSID of attached radio. Return string."""
+        device = self.radio.agent.device
+        device.sendline("iw {}.0 info".format(self.radio.iface_name))
```

Comment:
> Commit message says that you get it from the log file...

PSK for AP is received from the log, but not the SSID. I should recheck commit message if it states otherwise.